src/main/java/uk/gov/hmcts/reform/pip/account/management/service/subscription/UserSubscriptionService.java

```diff
@@ -50,8 +51,8 @@ public UserSubscription findByUserId(String userId) {
      * @return A confirmation message.
      */
     public String deleteAllByUserId(String userId) {
-        subscriptionListTypeRepository.deleteByUserId(userId);
-        subscriptionRepository.deleteAllByUserId(userId);
+        subscriptionListTypeRepository.deleteByUserId(UUID.fromString(userId));
```

Comment:
Is there a way of changing the method signature here to accept a UUID instead of a String similar to the findByUserId one above? Would prevent the need to cast

Reply:
Thanks for spotting this. I remembered that I missed something during the development but was unable to figure it out :)